css/style.css

```diff
@@ -474,8 +474,6 @@ template {
 .header__language-selector {
   margin-left: 10px;
   margin-right: 5px;
-}
-.language-selector {
```

Comment:
Why you decided to remove this class `language-selector`?

Reply:
Because this class don't use in html.

Reply:
Makes sense 🤔